db/Controllers/app.controllers.js

```diff
@@ -61,13 +60,18 @@ function addComment(req, res, next) {
   const { review_id } = req.params;
   const { username, body } = req.body;
 
-  const addUserPromise = addUser(username);
-  const addCommentPromise = addingComment(review_id, username, body);
+  const reviewIdPromise = fetchReview(review_id);
+  const addingCommentPromise = addingComment(review_id, username, body);
 
-  Promise.all([addUserPromise, addCommentPromise]).then((result) => {
-    const addComment = result[1];
-    res.status(201).send({ addComment });
-  });
+  Promise.all([reviewIdPromise, addingCommentPromise])
```

Comment:
ReviewIdPromise unnecessary, SQL will check foreign keys for us and return an error. Have a look at the seed file to see which these are if your unsure.